src/main/java/com/untamedears/JukeAlert/model/LoggedAction.java

```diff
@@ -59,4 +65,30 @@ public static LoggedAction getFromId(int id) {
             default: return UNKNOWN;
         }
     }
+    
+    /**
+     * Returns the 'human-friendly' version of the action
+     * Currently this is only used when displaying snitch logs
+     * @return
+     */
+    public String toActionString(){
```

Comment:
So returning here. Instead of this pattern which is OK if you have only a single "kind" of data, instead use an enum as a sort of class using the custom constructor:

KILL ("Killed", ChatColor.DARK_RED, 3),
BLOCK_PLACE ("Block Place", ChatColor.DARK_RED, 2),
etc.
w/ constructor
LoggedAction (String, ChatColor, int) {}

and private member attributes of String, ChatColor, and int -- and something like toActionString() just becomes `return this.actionString;` etc.

Cleaner, and lets you get rid of all these switching blocks entirely.